Entities/SceneLayer.cpp

```diff
@@ -285,24 +286,23 @@ namespace RTE {
 
 		m_BitmapClearMutex.lock();
 		if (m_LastClearColor != clearTo) {
-			// Damnit, our backbuffer is the wrong colour. Fix it up!
```

Comment:
I felt like this comment was important to clarify that it’s an exceptional case. If performance issues arose due to misuse of this tracked draw system, and this function was showing any significant CPU time spent in clear_to_color in the main thread, it would be very obvious to me that something has gone wrong with that comment. Whereas it might seem like an expected situation without it.

Reply:
Ahhh, I didn't realize the comment meant it was an exceptional case.
I don't disagree with having a comment explaining this is exceptional, but I don't think the comment made did that clearly. To me it just read as a dancomment:tm: that describes the code that's running.

Reply:
Fair enough! I thought the "Damnit" made it clear, but I suppose not - so yeah, a less tongue-in-cheek comment would work

Reply:
:+1:
Wanna put that into place?